app/config/locale/templates/email-smtp-test.tpl

```diff
@@ -0,0 +1,12 @@
+<p style="margin-block-start: 0;">Hello,</p>
+
+<p>This email ensures your custom SMTP configuration is working correctly. Please review your sender Information details:</p>
+
+<p style="margin-block-end: 0;">From: <strong>{{from}}</strong></p>
+<p style="margin-block-start: 0;">Reply-To: <strong>{{replyTo}}</strong></p>
+
+<p>If this email landed in your spam folder, ensure your SMTP server is set up correctly. A common mistake is overlooking some DNS configuration.</p>
```

Comment:
Your SMTP provider usually has checks that you need to satisfy. We cant link since we don't know who your SMTP provider is.

Here is example:

![CleanShot 2024-01-18 at 11 11 11@2x](https://github.com/appwrite/appwrite/assets/19310830/a79049aa-5b74-4804-af33-f7ddb109bf74)


SPF, DKIM, MX, .. Its not our expertise, I don't feel confident giving exact recommendations. SMTP provider should have you covered.

Reply:
I think you might also land in spam because of missing "Unsubscribe" button in newsletters, having spam-report rate above 0.5%, and similar. Sometimes even SMTP provider itself can block you to protect themselves. Same applies, your SMTP provider should help you get it back on track.

Reply:
Something like, feel free to change up details.
"If this email landed in your spam folder, please check your SMTP server settings to ensure they meet your provider's requirements. Often, missing DNS configurations or security checks mandated by your SMTP provider affect email delivery."

Reply:
Updated.